tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java

```diff
@@ -924,7 +926,7 @@ public void TestSwissHighGerman() {
         }
     }
 
-    public void TestExtraPaths() {
+    public void TestExtraPathsCapitalT() { // CapitalT to distinguish from testExtraPaths above
```

Comment:
Minor, but better as TestExtraPaths2 to distinguish. The CapitalT is unexpected.

Reply:
You're right. I've just changed it to `testExtraPaths2`. I guess I was unconsciously expressing some frustration about the existence of two functions, testExtraPaths and TestExtraPaths. The convention is for all Java methods to start with a lowercase letter. I don't know why test functions would be exceptions to that rule; maybe tech debt. In TestFmwk.java we have:

`if (name.startsWith("Test") || name.startsWith("test")) {`